includes/wikia/services/templates/Pagination_index.php

```diff
@@ -1,26 +1,26 @@
-<ul class="Pagination"  <? foreach($data as $key => $val ): ?> data-<?php echo $key ?>="<?php echo htmlentities($val); ?>" <? endforeach; ?> >
+<ul class="Pagination"  <? foreach($data as $key => $val ): ?> data-<?= htmlspecialchars( $key ) ?>="<?php echo htmlentities($val); ?>" <? endforeach; ?> >
```

Comment:
Is the `$key` in the users of this class influenced by users at all? If so, should we (in addition to escaping) validate that `$key` is a valid attribute name (no spaces or strange characters) since `htmlspecialchars` won't escape spaces and so forth? i.e. something like `Sanitizer::validateTagAttributes`?